src/styles/partials/_functions.scss

```diff
@@ -1,7 +1,7 @@
 // Custom functions
 
 @import 'vars';
-@import '~modularscale-sass';
+@import '../../../node_modules/modularscale-sass/stylesheets/modularscale';
```

Comment:
Couldnt get it to work without absolute path. Had tried different ways and the one from official readme.

Reply:
Hmm thinking if we are transpiling modules notation and it's causing issue, though I think we would see other issues.